ksnp4/4.0/kSNP_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+wget https://sourceforge.net/projects/ksnp/files/Examples.zip/download
+mv download Examples.zip
+unzip Examples.zip
+./RunExamples.sh
```

Comment:
Notes to self as to what this test does : 

This command downloads some examples files from sourceforge, renames the file to Examples.zip, decompresses that file, and then runs `RunExamples.sh`

I think this is a great test.